HPXMLtoOpenStudio/measure.rb

```diff
@@ -474,6 +501,82 @@ def self.set_defaults_and_globals(runner)
       @hpxml.misc_loads_schedule.monthly_multipliers = '1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248'
     end
 
+    # Default clothes washer location, rated annual kWh, label electric rate, label gas rate, label annual gas cost, capacity, and modified energy factor
+    if @hpxml.clothes_washers.size > 0
+      clothes_washer = @hpxml.clothes_washers[0]
+      if clothes_washer.location.nil?
+        clothes_washer.location = HPXML::LocationLivingSpace
+      end
+      if clothes_washer.rated_annual_kwh.nil?
+        clothes_washer.rated_annual_kwh = HotWaterAndAppliances.get_clothes_washer_reference_ler()
+        clothes_washer.label_electric_rate = HotWaterAndAppliances.get_clothes_washer_reference_elec_rate()
+        clothes_washer.label_gas_rate = HotWaterAndAppliances.get_clothes_washer_reference_gas_rate()
+        clothes_washer.label_annual_gas_cost = HotWaterAndAppliances.get_clothes_washer_reference_agc()
+        clothes_washer.capacity = HotWaterAndAppliances.get_clothes_washer_reference_cap()
+        clothes_washer.modified_energy_factor = HotWaterAndAppliances.calc_clothes_washer_mef_from_imef(HotWaterAndAppliances.get_clothes_washer_reference_imef())
+      end
+    end
+
+    # Default clothes dryer location, control type, and energy factor
+    if @hpxml.clothes_dryers.size > 0
+      clothes_dryer = @hpxml.clothes_dryers[0]
+      if clothes_dryer.location.nil?
+        clothes_dryer.location = HPXML::LocationLivingSpace
+      end
+      if clothes_dryer.control_type.nil?
+        clothes_dryer.control_type = HotWaterAndAppliances.get_clothes_dryer_reference_control()
+      end
+      if clothes_dryer.energy_factor.nil? && clothes_dryer.combined_energy_factor.nil?
+        clothes_dryer.energy_factor = HotWaterAndAppliances.calc_clothes_dryer_ef_from_cef(HotWaterAndAppliances.get_clothes_dryer_reference_cef(clothes_dryer.fuel_type))
```

Comment:
I would think these are probably linked properties and cannot be independently defaulted (similar to clothes washers).